frontend/features/page/sections/ContactsSection/ContactsSection.tsx

```diff
@@ -0,0 +1,517 @@
+                }
+              : undefined
+          }>
+          <InlineTextarea name="sectionTitle" />
+        </Box>
+
+        <Flex
+          flexDir={{
+            base: "column",
+            md: "row",
+          }}>
+          <Flex
+            flex={{
+              base: "0 0 full",
+              md: "0 0 300px",
+              xl: "0 0 400px",
+            }}
+            mr={{
+              base: "0",
+              md: "20",
+              lg: "36",
+            }}
+            flexDir="column">
+            <Box
+              as="h2"
+              fontSize={{ base: "3xl", sm: "4xl", md: "5xl" }}
+              fontFamily="Europa"
+              fontWeight="bold"
+              lineHeight="1.1em">
+              <InlineTextarea name="title" />
+            </Box>
+            <Text fontSize="sm" pt="5" color="description">
+              <InlineTextarea name="subtitle" />
+            </Text>
+            <Link href="mailto:DA INSERIRE" passHref>
+              <Box
+                as="a"
+                fontSize="sm"
+                pt="14"
+                lineHeight="1.8em"
+                textDecoration="none">
+                <InlineTextarea name="email" />
+              </Box>
+            </Link>
+          </Flex>
+          <ContactsForm />
+        </Flex>
+
+        <Flex justifyContent="flex-end" w="full" mb="32">
+          <Social
+            isBubbleActive={areBubblesActive}
+            link="https://twitter.com/inkofpixel"
+            color="rgba(29, 161, 242, 0.7)"
+            onHoverColor="rgba(29, 161, 242, 1)">
+            <TwitterIcon color="white" />
+          </Social>
+          <Social
+            isBubbleActive={areBubblesActive}
+            link="https://facebook.com/inkofpixel"
+            color="rgba(59, 89, 152, 0.7)"
+            onHoverColor="rgba(59, 89, 152, 1)">
+            <FacebookIcon color="white" />
+          </Social>
+          <Social
+            isBubbleActive={areBubblesActive}
+            link="https://github.com/inkofpixel"
+            color="rgba(24, 23, 23, 0.7)"
+            onHoverColor="rgba(24, 23, 23, 1)">
+            <GithubIcon color="white" />
+          </Social>
+        </Flex>
+      </Flex>
+    </Box>
+  );
+}
+
+const blankForm = {
+  name: "",
+  email: "",
+  message: "",
+  isSubmitted: false,
+};
+
+function reducer(state: any, action: any) {
+  switch (action.type) {
+    case "setName":
+      return {
+        ...state,
+        name: action.value,
+      };
+    case "setEmail":
+      return {
+        ...state,
+        email: action.value,
+      };
+    case "setMessage":
+      return {
+        ...state,
+        message: action.value,
+      };
+    case "setIsSubmitted":
+      return {
+        ...state,
+        isSubmitted: !state.isSubmitted,
+      };
+  }
+}
+
+function ContactsForm() {
+  const [state, dispatch] = React.useReducer(reducer, blankForm);
+
+  async function sendMessage(event: Event) {
```

Comment:
we usually prefer event handlers to be call with `on..`, so for example `onFormSubmit`.

Also Event is not the right type here.. You should probably use something like `React.FormEvent<..>`